AdblockPlusSafari/AppDelegate.m

```diff
@@ -20,14 +20,27 @@
 #import "Appearence.h"
 #import "RootController.h"
 
+// Update filter list every 5 days
+const NSTimeInterval FilterlistsUpdatePeriod = 3600*24*5;
+// Wake up application every 12 hours (just hint for iOS)
+const NSTimeInterval BackgroundFetchInterval = 3600*12;
```

Comment:
It seems to me that this value is completely ignored by iOS. On my phone it wake up app every six hours regardless value I set here.